### PR TITLE
Improve error info in real-aws integration test script.

### PR DESCRIPTION
**Pull request by fisx**
This worked for me.  Thanks @tiago-loureiro!

Reviewer: please double-check the aws-cli github repo and the s3 path.

### CODE REVIEW

File: services/gen-aws-conf.sh [hunk 1]
```diff
@@ -4,7 +4,20 @@ set -e
...
+      exit 22 \
+    )
+
+command -v awscli >/dev/null || \
+    ( echo "*** please install awscli ( https://github.com/aws/aws-cli ) in your path."; \
+      echo "    e.g.: pip install awscli"; \
+      exit 22 \
+    )
+
+echo -e "\nHINT: when prompted for an integration config, consider trying 'z-config/integration/integration-aws.yaml'.\n"
```
Comment (neongreen) [inline services/gen-aws-conf.sh]: Perhaps give a command to download this?

Comment (fisx) [reply services/gen-aws-conf.sh]: No, you just have to type in that path.  The download is done by the script.

File: services/gen-aws-conf.sh [hunk 2]
```diff
@@ -4,7 +4,20 @@ set -e
...
 
-which yq >/dev/null || ( echo "*** please install yq ( https://github.com/mikefarah/yq ) in your path."; exit 22 )
+command -v yq >/dev/null || \
+    ( echo "*** please install yq ( https://github.com/mikefarah/yq ) in your path."; \
+      echo "    e.g.: wget https://github.com/mikefarah/yq/releases/download/2.1.2/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq"; \
+      echo "    NOTE: later versions may not work."; \
+      exit 22 \
+    )
+
+command -v awscli >/dev/null || \
```
Comment (neongreen) [inline services/gen-aws-conf.sh]: Shouldn't this be just `aws`? At least on my machine it's `aws`.

Comment (fisx) [reply services/gen-aws-conf.sh]: you're right.  thanks!

**Pull request merged**